ci-operator/config/openshift/installer/openshift-installer-main.yaml

```diff
@@ -1497,10 +1497,18 @@ tests:
   run_if_changed: (gcp|google)
   steps:
     cluster_profile: gcp
-    env:
-      FEATURE_GATES: '["GCPCustomAPIEndpointsInstall=true"]'
-      FEATURE_SET: CustomNoUpgrade
     workflow: openshift-e2e-gcp-custom-endpoints
+- always_run: false
+  as: e2e-gcp-custom-endpoints-proxy-wif
+  optional: true
+  run_if_changed: (gcp|google)
+  steps:
+    cluster_profile: gcp
+    env:
+      ADDITIONAL_WHITELIST_URLS: .amazonaws.com
```

Comment:
From the past job logs, the address being accessed is `s3.us-east-1.amazonaws.com`. 
I think we can directly specify this address instead of `.amazonaws.com`, as `.amazonaws.com` covers too many addresses.